### ISSUE TITLE
Error graph misleading ?

### ISSUE THREAD

**Issue by kbenahmed89** [vlfeat/matconvnet#478]
Hi,

I'm tuning a pre-trained network, I extract features from the network after every iteration and classify my images using RandomForest. The results I get are completely different than what the error graph I generated during training shows. In the error graph the accuracy starts low and then it gets better at the end. This is the error graph I got :
![image](https://cloud.githubusercontent.com/assets/17075128/14095303/dca51424-f52c-11e5-8457-828401501e2d.png)

the results in RandomForest by iteration are:

![image](https://cloud.githubusercontent.com/assets/17075128/14095314/f9fa9152-f52c-11e5-8110-183dc4f552b9.png)

how do you explain this please??

thank you.

**Comment by bazilas** [comment]
Do you use the same error metric in both cases (because it doesn't look likes this)?

Also you shouldn't expect exactly the same performance from the sofmax classifer (neural net) and the random forest.
